backend/utils/ping_client.py

```diff
@@ -0,0 +1,49 @@
+import time
+import json
+from game_flow.lobby_repository import LobbyRepository
+
+lobby_repository = LobbyRepository()
+
+
+def ping_client(websocket, player_id, game_id):
+    while True:
+        time.sleep(2)
+        try:
+            ping_message = json.dumps({"method": "ping"})
+            websocket.send(ping_message)
+            websocket.ping_time = time.time()
+
+            time.sleep(2)
+            if not websocket.pong_received:
```

Comment:
how will this one be false?

Reply:
Oops I did some refactoring after I made the video demo and must've of accidentally removed it. Same with the comment above, that one should've been removed instead

Reply:
Fixed